### PR TITLE
Enable the user to control which external images not to display

### PR DESCRIPTION
**Pull request by Chris00**

### CODE REVIEW

**Comment by Chris00** [comment]
Here is a notification with an image so you can test! 😀
image

File: mu4e/mu4e-view-gnus.el [hunk 1]
```diff
@@ -37,6 +37,27 @@
...
 
+(defvar mu4e-view-blocked-images "."
+  "Regexp matching image URLs to be blocked, or nil meaning not
+to block.  Beware that loading external images may lower your privacy.
+
+This can also be a function that takes a message as argument and
+returns a regexp.  For example, if you want to see the all images
+in Github notifications (which contain no code to personally
+identify you, Github will only ba able to tell someone from your
+IP has acessed the image), you can set this variable to the
```
Comment (djcb) [inline mu4e/mu4e-view-gnus.el:48]: "be able"

File: mu4e/mu4e-view-gnus.el [hunk 2]
```diff
@@ -37,6 +37,27 @@
 (defvar helm-comp-read-use-marked)
 (defvar mu4e~view-rendering nil)
 
+(defvar mu4e-view-blocked-images "."
+  "Regexp matching image URLs to be blocked, or nil meaning not
```
Comment (djcb) [inline mu4e/mu4e-view-gnus.el:41]: Can you make it a defcustom, mirroring the gnus one?

**Review by djcb — COMMENTED** [review]
Looks good (with the few changes mentioned).

File: mu4e/mu4e-view-gnus.el (hunk 2 above)
Comment (Chris00) [reply mu4e/mu4e-view-gnus.el:41]: Done.

**Comment by Chris00** [comment]
Fixed a typo in the last push.

**Comment by djcb** [comment]
Merged -- thanks!

**Pull request merged**